### PR TITLE
Add annotations to README title

### PR DESCRIPTION
**Pull request by xtydev**
for the avoidance of doubt and misunderstanding, and to make the title more descriptive of the participatory nature of the project

### CODE REVIEW

**Comment by xtydev** [comment]
As can be seen from the [__Issues__](https://github.com/MSWorkers/support.996.ICU/issues) section, a substantial number of supporters think of this project as a statement put out by Microsoft. #110 references some of the confusion and misplaced gratitude.

This PR serves to give a more accurate impression of the aims and objectives of this project, and remind people of its participatory nature.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
-# Microsoft and GitHub Workers Support 996.ICU
-# 微软员工和 GitHub 员工宣布支持 996.ICU 运动
+# Support 996.ICU
```
Comment (MSWorkers) [inline README.md]: Please keep the title of the letter the same

**Comment by MSWorkers** [comment]
@xtydev thank you for bringing this up. For now I think we should not change the title of the letter. I think we could just add a note at the top of the letter:
> note: etc...

**Comment by xtydev** [comment]
Done! I agree that for now we need a sensational title to maximize the impact of this project.

**Comment by MSWorkers** [comment]
@xtydev one last nitpick - can we change "Join forces with Microsoft and Github workers..." to "Join Microsoft and Github workers..."

**Comment by xtydev** [comment]
@MSWorkers Done. Brevity for the win!

**Pull request merged**